testing/scenario_app/ios/Scenarios/ScenariosUITests/iPadGestureTests.m

```diff
@@ -196,6 +199,99 @@ - (void)testPointerHover {
   XCTAssertTrue([app.textFields[removeMessage] waitForExistenceWithTimeout:1],
                 @"PointerChange.remove event did not occur for a hover");
 }
+
+- (void)testPointerScroll {
+  BOOL supportsPointerInteraction = NO;
+  SEL supportsPointerInteractionSelector = @selector(supportsPointerInteraction);
+  if ([XCUIDevice.sharedDevice respondsToSelector:supportsPointerInteractionSelector]) {
+    supportsPointerInteraction =
+        performBoolSelector(XCUIDevice.sharedDevice, supportsPointerInteractionSelector);
+  }
+  XCTSkipUnless(supportsPointerInteraction, "Device does not support pointer interaction.");
+  XCUIApplication* app = [[XCUIApplication alloc] init];
+  app.launchArguments = @[ @"--pointer-events" ];
+  [app launch];
+
+  NSPredicate* predicateToFindFlutterView =
+      [NSPredicate predicateWithFormat:@"identifier BEGINSWITH 'flutter_view'"];
+  XCUIElement* flutterView = [[app descendantsMatchingType:XCUIElementTypeAny]
+      elementMatchingPredicate:predicateToFindFlutterView];
+  if (![flutterView waitForExistenceWithTimeout:kSecondsToWaitForFlutterView]) {
+    NSLog(@"%@", app.debugDescription);
+    XCTFail(@"Failed due to not able to find any flutterView with %@ seconds",
+            @(kSecondsToWaitForFlutterView));
+  }
+
+  XCTAssertNotNil(flutterView);
+
+  SEL scroll = @selector(scrollByDeltaX:deltaY:);
+  XCTAssertTrue([flutterView respondsToSelector:scroll],
+                @"If supportsPointerInteraction is true, this should be true too.");
+  // Need to use NSInvocation in order to send primitive arguments to selector
+  NSInvocation* invocation = [NSInvocation
+      invocationWithMethodSignature:[XCUIElement instanceMethodSignatureForSelector:scroll]];
+  [invocation setSelector:scroll];
+  CGFloat deltaX = 0.0;
+  CGFloat deltaY = 1000.0;
+  [invocation setArgument:&deltaX atIndex:2];
+  [invocation setArgument:&deltaY atIndex:3];
+  [invocation invokeWithTarget:flutterView];
+
+  // The hover pointer is removed after ~3.5 seconds, this ensures that all events are received
```

Comment:
Where is the 3.5 seconds defined?

Reply:
It is the observed behavior on iPadOS. After around 3.5 seconds of not moving the mouse, the system will remove it.

Reply:
This might be a risk of flaky test. It is possible to find a different way to test this?

Reply:
We could end the test without checking that the pointer is removed after a delay, since it's just an observed behaviour of the iPad, you're right that it could change with any update. The rest of the test can stay.